finetune/base_models/textcnn/model.py

```diff
@@ -0,0 +1,20 @@
+from finetune.base_models import SourceModel
+from finetune.base_models.gpt.encoder import GPTEncoder
+from finetune.base_models.textcnn.featurizer import textcnn_featurizer
+
+class TextCNNModel(SourceModel):
+
+    encoder = GPTEncoder
+    featurizer = textcnn_featurizer
+    kernel_sizes = [2, 3, 4]
+    settings = {
+        'n_embed_featurizer': 768,
```

Comment:
This should be made consistent across the other models, or a comment and exceptions to show, for example, that if they are set but are different, this cant work on the other models.

Reply:
Discussed offline, will deal with this in a future PR